### PR TITLE
SCP-3830: Trim serialiseData from v1-costmodel-params

### PR DESCRIPTION
**Pull request by bezirg**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [x] Self-reviewed the diff
    - [ ] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

**Comment by kwxm** [comment]
Eh?

File: plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/BuiltinCostModel.hs [hunk 1]
```diff
@@ -161,6 +161,15 @@ deriving via CustomJSON '[FieldLabelModifier (StripPrefix "param", LowerIntialCh
 deriving via CustomJSON '[FieldLabelModifier (StripPrefix "param", LowerIntialCharacter)]
              (BuiltinCostModelBase CostingFun) instance FromJSON (BuiltinCostModelBase CostingFun)
 
+-- | Same as 'CostingFun' but maybe missing.
+-- We could use 'Compose Maybe CostinFun' instead but we would then need an orphan ToJSON instance.
```
Comment (kwxm) [inline plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/BuiltinCostModel.hs:165]:
```suggestion
-- We could use 'Compose Maybe CostingFun' instead but we would then need an orphan ToJSON instance.
```

**Comment by michaelpj** [comment]
Maybe somehow something was happening on the benchmark machine?

**Review by michaelpj — APPROVED** [review]
Nice!

**Comment by michaelpj** [comment]
I have no idea what's going on with the benchmarks, but at least it's weirdly faster rather than weirdly slower, and we need this in.

**Comment by michaelpj** [comment]
@bezirg can you do the backport PR?

**Comment by bezirg** [comment]
> @bezirg can you do the backport PR?

on it

**Pull request merged**